src/api/app/views/webui2/webui/package/_bottom_actions.html.haml

```diff
@@ -4,7 +4,7 @@
     = render partial: 'webui2/webui/package/bottom_actions/bugzilla_owner', locals: { bugowners_mail: bugowners_mail,
                                                                 package_name: package.name, project_name: project.name }
   - unless User.current.is_nobody?
-    - if current_rev
+    - if current_rev && is_working
```

Comment:
I find `is_working` a little generic, but it'll do

Reply:
My point is exactly about the 'working' - there are many definitions of what 'package is working' means.

Reply:
aaah! :thinking: :thinking: :thinking: